code/modules/core_implant/core_implant.dm

```diff
@@ -127,7 +127,7 @@
 			if(R.ignore_stuttering)
 				ture_message = message_pre_problems
 			if(R.compare(ture_message))
-				if(R.power > src.power)
+				if(R.power > src.power && !(power >= max_power))
```

Comment:
This is problematic because it means litanies intended to only ordinarily be used by a Prime are now in the reach of everyone.

Reply:
The idea here is that a cruciform won't prevent someone from using a litany they have unlocked. If someone is given a litany it shouldn't sit around useless.

Reply:
Except that those litanies are deliberately designed that way, so that normal people can't used. They aren't "unlocked" unless you become a Crusader or a Prime, that is the design intent. Scry is way too powerful to have in everyone's hands. The only things that anyone can theoretically do but cannot because of their max power is the Episcopal litanies deliberately designed to be out of reach.

Reply:
Isnt it a church taboo/get your implant removed if you use scry without consent?
Also crusaders get a power boost to be able to use it
prime conversion kit dosnt, but that shouldnt be handed out carelessly ICly

Reply:
> Isnt it a church taboo/get your implant removed if you use scry without consent? Also crusaders get a power boost to be able to use it prime conversion kit dosnt, but that shouldnt be handed out carelessly ICly

I think something mentions it being considered rude, but it's not in any way forbidden. And yes, crusaders and primes (the kit *does* give you a power boost) do get access to it, but those are rare. Crusaders are admin only, and other than the starting Prime you have to go through the process to consecrate a new one, which rarely happens.

Reply:
So, are there any litanies that someone is currently unable to use because of power limitations and not because they're prevented from using it by their cruciform's module?

Reply:
Yes, but *only by design*. There are two litanies that are soft-locked to primes and crusaders by a power cap higher than Vectors can get: Scrying and Consecration. If this were to be put through, I would need to then make a PR that actually module locks them because they are too powerful to be in the hands of normal Vectors, thus rendering this PR moot. Additionally, negative power just makes no fucking sense.

Reply:
Other than those two, there is no way to have a litany that is available to your module but inaccessible for power reasons.